src/main/java/mops/foren/infrastructure/web/ForenController.java

```diff
@@ -26,6 +23,7 @@
     private ThreadService threadService;
     private PostService postService;
     private ForumForm form = new ForumForm("", "");
+    private PostForm postForm = new PostForm("");
```

Comment:
why are they empty?

Reply:
good point, to be decided in the future. In general, for all forms that we use.